data/pools/transactionPool.go

```diff
@@ -164,7 +164,12 @@ func (pool *TransactionPool) rememberCommit(flush bool) {
 func (pool *TransactionPool) PendingCount() int {
 	pool.pendingMu.RLock()
 	defer pool.pendingMu.RUnlock()
+	return pool.pendingCountLocked()
+}
 
+// pendingCountLocked is a helper for PendingCount that returns the number of
+// transactions pending in the pool
+func (pool *TransactionPool) pendingCountLocked() int {
```

Comment:
the name confusing, please rename to `pendingCountNoLock` or similar

Reply:
updated, thanks!